tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_spme.py

```diff
@@ -124,6 +124,15 @@ def test_surface_form_algebraic(self):
         modeltest = tests.StandardModelTest(model)
         modeltest.test_all()
 
+    def test_integrated_conductivity(self):
+        model = pybamm.lithium_ion.SPMe(build=False)
+        model.submodels[
+            "electrolyte conductivity"
+        ] = pybamm.electrolyte_conductivity.Integrated(model.param)
+        model.build_model()
```

Comment:
You might want to provide an option for doing this automatically

Reply:
I thought about that, but wasn't sure what approach we want to take. We discussed it some time ago, but can't remember what were the arguments against being able to pass options to set all the submodels.

Reply:
I don't remember the arguments against either. I think you should add the option